packages/backend/src/db/house/houseStore.service.ts

```diff
@@ -29,6 +30,23 @@ export class HouseStoreService {
     return this.HouseModel.find().exec();
   }
 
+  async getUserDto(
+    id: string | Mongoose.Types.ObjectId,
+  ): Promise<UserResponseDto[]> {
+    const house = await this.HouseModel.findOne({ _id: id })
+      .populate('users')
+      .exec();
+    const userList: Array<UserResponseDto> = [];
+    for (const user of house.users) {
+      const userDto = {
+        house: house.code,
+        firebaseId: user['firebaseId'],
+      };
+      userList.push(userDto);
+    }
+    return userList;
```

Comment:
Can you just return the `house.users`? If not, what is there is okay, but it would be simpler to return `house.users`.